juriscraper/opinions/united_states/state/colo.py

```diff
@@ -7,20 +7,26 @@
 Date created: 2016-06-03
 """
 
+import re
 import certifi
 import requests
 from lxml import html
 
 from juriscraper.AbstractSite import logger
 from juriscraper.OpinionSite import OpinionSite
 from juriscraper.lib.string_utils import convert_date_string
+from juriscraper.AbstractSite import InsanityException
 
 
 class Site(OpinionSite):
     cases = []
     sub_page_opinion_link_path = "//a[@class='Head articletitle']"
     parent_summary_block_path = 'parent::p/following-sibling::div[@class="Normal"][1]'
 
+    # I'm certain this can be done more professionally,
+    # but I (arderyp) am not gifted at the art of regex
+    regex = '(\d+)\s+(\w+)\s+(\d+\.?)\s+((Nos?\.?\s+)?((\w{5,8}\.?)(((\s+\&|\,)\s+\w{5,8})+)?))\.?(\s+)?(.*)'
```

Comment:
Oh, I don't know. It's explicit, which is a lot more than I can say for most regexes...